packages/dd-trace/src/profiling/profilers/cpu.js

```diff
@@ -0,0 +1,53 @@
+'use strict'
+
+class NativeCpuProfiler {
+  constructor (options = {}) {
+    this.type = 'wall'
+    this._samplingInterval = options.samplingInterval || 10 * 1000
+    this._mapper = undefined
```

Comment:
What's the benefit of manually re-assigning `undefined`?

Reply:
Assigning it to `undefined` in the constructor avoids the [hidden class](https://mrale.ph/blog/2012/06/03/explaining-js-vms-in-js-inline-caches.html) problem. It's a performance optimization.